### PR TITLE
Add rules C409 and C410 that complain about unnecessary list/tuple

### PR DESCRIPTION
**Pull request by jdufresne**

### CODE REVIEW

File: flake8_comprehensions.py [hunk 1]
```diff
@@ -27,6 +27,8 @@ def __init__(self, tree, *args, **kwargs):
         'C405': 'C405 Unnecessary {type} literal - rewrite as a set literal.',
         'C406': 'C406 Unnecessary {type} literal - rewrite as a dict literal.',
         'C407': "C407 Unnecessary list comprehension - '{func}' can take a generator.",
+        'C408': 'C408 Unnecessary {type} literal - rewrite as a tuple literal.',
+        'C409': 'C409 Unnecessary {type} literal - rewrite as a list literal.',
```
Comment (pope1ni) [inline flake8_comprehensions.py]: I think that these need rewording? Perhaps:

```diff
+        'C408': 'C408 Unnecessary {type} literal passed to tuple constructor - rewrite as a tuple literal.',
+        'C409': 'C409 Unnecessary {type} literal passed to list constructor - rewrite as a list literal.',
```

Maybe a little wordy and perhaps "constructor" isn't the best term, but should be clearer...

Comment (adamchainz) [inline flake8_comprehensions.py]: "passed to tuple()" ?

**Comment by jdufresne** [comment]
Thanks for the review. I have updated the wording as suggested.

**Comment by adamchainz** [comment]
Can you rebase, now I've merged #51 this needs the error numbers C409 and C410. Additionally when hitting C410 we'll need the entry point in `setup.py` to be updated from `C40 =` to allow for the new code - I'm not sure if there's an easy way of specifying C40X and C41X are registered by the plugin 🤔 .

**Comment by jdufresne** [comment]
Rebased and adjusted the entry point. Thanks.

File: README.rst [hunk 2]
```diff
@@ -106,14 +108,3 @@ The list of builtins that are checked for are:
 * ``sorted``
 * ``sum``
 * ``tuple``
-
-C408: Unnecessary (dict/list/tuple) call - rewrite as a literal.
```
Comment (pope1ni) [inline README.rst]: I presume that this removal is unintentional after the rebase? Should a section for C409-C410 be added?

Comment (adamchainz) [inline README.rst]: Indeed it should!

**Comment by jdufresne** [comment]
Thanks for the review. Good catch. I have updated the commit:

* Restored deleted README section
* Added details to README about the new rules

Cheers.

**Comment by jdufresne** [comment]
@adamchainz Do you feel your comment in #63 applies here as well. "rewrite as list literal" may be confusing as there is already a list literal inside the `list()` call.

**Comment by adamchainz** [comment]
Good point, of course you're also detecting when there's a list literal inside a call to `tuple`, so to do it clearly you might need some logic around the message. Shouldn't be that bad though 🤔

**Comment by jdufresne** [comment]
Updated with suggested changes.

**Pull request merged**